tools/circt-test/CMakeLists.txt

```diff
@@ -29,5 +29,13 @@ target_link_libraries(circt-test PRIVATE ${libs})
 llvm_update_compile_flags(circt-test)
 mlir_check_all_link_libraries(circt-test)
 
-configure_file(circt-test-runner-sby.py ${CIRCT_TOOLS_DIR}/circt-test-runner-sby.py)
-add_custom_target(circt-test-runner-sby SOURCES ${CIRCT_TOOLS_DIR}/circt-test-runner-sby.py)
+foreach(runner sby circt-bmc)
+  configure_file(
+    circt-test-runner-${runner}.py
+    ${CIRCT_TOOLS_DIR}/circt-test-runner-${runner}.py
+  )
```

Comment:
Nit: Using `configure_file` adds a dependency between this source file and the configuration step, i.e., whenever the python file is modified, this cmake step is run again instead of just copying the file. I don't think this is necessary.
I *think* the way to avoid this is to use `add_custom_command`. You can take a look at how this is done for the python bindings:
https://github.com/llvm/llvm-project/blob/83c7784c35918ce037823f29d29918c5542cdf9c/mlir/cmake/modules/AddMLIRPython.cmake#L598-L605